test-all-commands.sh

```
@@ -1,65 +1,191 @@
+
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)